README.md

```diff
@@ -1,2 +1,210 @@
-# Practice-React
-This repo is created for learning React basic and gaining knowledge on how to give code review.
+<a name="readme-top"></a>
+
+<div align="center">
+ <h3><b>practice-react</b>.
+ </h5>
+</div>
+
+<div align="center">
+<h3><b>screenshot</b></h3>
+It will be added when the project is completed.
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo and presentation video](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 📖 practice-react <a name="about-project"></a>
+
+<h5> I set up the environment and tools needed to develop a React application. In this project, I have made a simple SPA To-Do-App project that allows user to add and remove To Do and display on UI .</h5>
+
+## 🛠 Built With <a name="built-with"></a>
+
+## Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <b>This project only works client side right now.</b>
+  <ul>
+    <li>JSX</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>React</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href=#>Local Server - It will be Hosted on Github when finish</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href=#>I will us API's in future</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+## Key Features <a name="key-features"></a>
+
+- Add To-Do Items
+- Remove To-Do Items from list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo<a name="live-demo"></a>
+
+- [Live Demo Link](https://feature-2--sparkling-dieffenbachia-24a95b.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).


- For learning purposes, kindly use the GitHub pages for live deployment

Reply:
### Hello @Santosh-Konappanavar,
Thanks for your valuable insights. Maybe there are no rules about it and I am already familiar with using the gh-page. I added the Netlify link because I am new to using Netlify.